test/Audio/SoundStream.test.cpp

```diff
@@ -1,9 +1,24 @@
 #include <SFML/Audio/SoundStream.hpp>
 
+#include <catch2/catch_test_macros.hpp>
+
 #include <type_traits>
 
-static_assert(!std::is_constructible_v<sf::SoundStream>);
-static_assert(!std::is_copy_constructible_v<sf::SoundStream>);
-static_assert(!std::is_copy_assignable_v<sf::SoundStream>);
-static_assert(!std::is_nothrow_move_constructible_v<sf::SoundStream>);
-static_assert(!std::is_nothrow_move_assignable_v<sf::SoundStream>);
+TEST_CASE("[Audio] sf::SoundStream")
+{
+    SECTION("Type traits")
+    {
+        STATIC_CHECK(!std::is_constructible_v<sf::SoundStream>);
+        STATIC_CHECK(!std::is_copy_constructible_v<sf::SoundStream>);
+        STATIC_CHECK(!std::is_copy_assignable_v<sf::SoundStream>);
+        STATIC_CHECK(!std::is_nothrow_move_constructible_v<sf::SoundStream>);
+        STATIC_CHECK(!std::is_nothrow_move_assignable_v<sf::SoundStream>);
+    }
+
+    SECTION("Chunk")
+    {
+        const sf::SoundStream::Chunk chunk;
+        CHECK(chunk.samples == nullptr);
+        CHECK(chunk.sampleCount == 0);
```

Comment:
Now that `sf::SoundStream::Chunk` has initial values we can write tests for those initial values. Because these tests happen at runtime, we needed to create a `TEST_CASE` for `sf::SoundStream` which didn't already have. I thought it made sense to include this in the same commit as the other changes.